homeassistant/components/screenlogic/__init__.py

```diff
@@ -140,17 +138,34 @@ def __init__(self, hass, *, config_entry, gateway):
             ),
         )
 
+    @property
+    def gateway_data(self) -> dict[str | int, Any]:
+        """Return the gateway data."""
+        return self.gateway.get_data()
```

Comment:
Note that `get_data()` is just returning a `dict` from the lib and is not doing any I/O

[dieselrabbit/screenlogicpy@`v0.6.4...v0.7.0`#diff-5d2d9004af](https://github.com/dieselrabbit/screenlogicpy/compare/v0.6.4...v0.7.0#diff-5d2d9004aff8986347d2b2c7683b2a51a7577fc59caf9e90c5ef30d96adf037bR154)